Source/Core/DolphinQt/Debugger/CodeDiffDialog.cpp

```diff
@@ -428,7 +549,9 @@ void CodeDiffDialog::InfoDisp()
          "list "
          "and any includes left over will be displayed.\nYou can continue to use "
          "'Code did not get executed'/'Code has been executed' to narrow down the "
-         "results."));
+         "results.\n\n"
+         "Saving will store the current list in:\n") +
+          QString::fromStdString(File::GetUserPath(D_LOGS_IDX)));

```

Comment:
Right now this returns the absolute path, so you could end up with a huge string. I recommend starting with `\User` path (do a split on it)
![image](https://github.com/dolphin-emu/dolphin/assets/14857235/f9389c44-6dda-43ca-98c7-1f62ce290c2c)

Reply:
Incase there's concern about the "where is the user folder", Dolphin has that built in.
![image](https://github.com/dolphin-emu/dolphin/assets/14857235/d9f43a8c-7a29-44fa-9a9b-a699134f0e92)

Reply:
Not sure everyone has User in the path, certainly not near the end unless it's in portable mode.  I could say it's in the Logs folder and reference the option to open the user folder.

Reply:
'User Folder' is the name of the folder. Whether it ends up being the Document \ Dolphin Emulator or the AppData\Dolphin, I think its still known as 'User Folder'.

Your proposal would work as well, I think if someone is using this tool they know about \Maps, they can probably find the \Logs.